index.html

```diff
@@ -181,6 +181,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var fireballSize = 22;
+
+  function getFireballSpeed (isWindFromLeft) {
+    if (isWindFromLeft) {
+      return 5;
+    }
+
+    return 2;
```

Comment:
```suggestion
    return isWindFromLeft ? 5 : 2
```
доп критерий https://up.htmlacademy.ru/javascript/20/criteries#d17